codeflash/api/cfapi.py

```
@@ -66,9 +72,10 @@ def make_cfapi_request(
         except (ValueError, TypeError):
             error_message = response.text
 
-        logger.error(
-            f"CF_API_Error:: making request to Codeflash API (url: {url}, method: {method}, status {response.status_code}): {error_message}"
-        )
+        if not suppress_errors:
```

Q: Why are we suppressing the errors?